registry/modules/upload-to-s3/utils/errors/storeFileS3.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpError } from "@/modules/shared/utils/errors/HttpError.js";
+
+export const fileUploadFailed = (errorCode?: string) => {
+  return new HttpError(
+    "FileUploadFailed",
+    `There was an error uploading your file. ${
+      errorCode && "Error code: " + errorCode
+    }`,
+    400
+  );
+};
+
+export const s3UploadFailed = () => {
+  return new HttpError(
+    "S3FileUploadFailed",
+    `There was a problem when uploading your file to your S3 storage. Please check logs and try again`,
+    400
+  );
+};
+
+export const errorDownloadingS3File = (fileName: string) => {
+  return new HttpError(
+    "ErrorDownloadingS3File",
+    `There was a problem when trying to download file ${fileName} from your S3 storage`,
+    404
```

Comment:
404 means Not Found. If the file does exist but downloading fails for an other reason, then I dont think 404 is a good status code.